src/lambda_codebase/initial_commit/bootstrap_repository/adf-bootstrap/deployment/lambda_codebase/initial_commit/pipelines_repository/example-deployment_map.yml

```diff
@@ -59,6 +59,18 @@ pipelines:
       - path: /banking/testing
         name: fancy-name
 
+  - name: sample-ecs-app
+    default_providers: # if we omit build and deploy type we get a default of codebuild as the build and cloudformation as the deploy. if using codecommit, account_id is required
+      source:
+        provider: codestar
+        properties:
+          repository: my-ecs-app # Optional, above name property will be used if this is not specified
+          owner: github-enterprise-team-org
+          codestar_connection_path: /ghes/connection1/arn # The path in AWS Systems Manager Parameter Store that holds the AWS CodeStar Connection arn
```

Comment:
This looks like a path used in an organization. Can you make it easier to read so people will understand they can change the value to their own preference? In its current form it looks like I cannot change it, while you can, am I right?

Reply:
yes, this is path is arbitrary. will update it to /path/to/parameter